### DISCUSSION TITLE
snitch v1.0.0 released

### DISCUSSION THREAD

**Discussion by cschreib** [snitch-org/snitch#58] [Announcements]
- The GitHub release, see full change log there: [link](https://github.com/cschreib/snitch/releases/tag/v1.0.0)
 - The reddit announcement: [link](https://www.reddit.com/r/cpp/comments/10dac2x/snitch_v10_a_lightweight_c20_testing_framework/)

Feel free to share feedback here or on reddit!

**Comment by willwray** [comment]
snitch gets a 2 minute mention on [cppcast](https://cppcast.com/val-and-mutable-value-semantics/)  Jan 20, 2023, starting after 6m30.
Phil 'Catch' Nash quickly reviews and Timur 'doctest-user' Doumler plans to give it a go.

**Reply by cschreib** [reply]
Thank you so much for letting me know; I thought cppcast had stopped airing, but I used to listen to it frequently. Very honoured to be mentioned there, especially given the participants :) That made my day.

**Comment by willwray** [comment]
Another mention on the lastest [cppcast](https://cppcast.com/cpp_in_finance/).
Phil still hasn't tried it...

**Reply by cschreib** [reply]
Super cool! Glad to heard people find the constexpr testing macros interesting.
